src/components/organisms/landingPageLinks/LandingPageLinks.tsx

```diff
@@ -0,0 +1,20 @@
+import { ImageList, useTheme, useMediaQuery } from '@mui/material'
+import { PageEnum } from '../../../common/constants/constants'
+import { EnumImageListItem } from '../../molecules/enumImageListItem/EnumImageListItem'
+
+type LandingPageLinksProps = {
+  pages: PageEnum[]
+}
+
+export function LandingPageLinks({ pages }: LandingPageLinksProps) {
+  const theme = useTheme()
+  const matches = useMediaQuery(theme.breakpoints.up('md'))
+  theme.breakpoints.up('md')
```

Comment:
This line looks redundant to me. Remove?

Reply:
Oh, yea. I was exploring the return. Left it in.

Reply:
```suggestion

```